util/rom-ext-manifest-generator.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+from pathlib import Path
+
+import hjson
+from mako.template import Template
+
+from topgen.c import MemoryRegion, Name
+
+DESC = """ROM_EXT manifest generator"""
+
+USAGE = """
+  rom-ext-manifest-generator --input-dir:
+    Directory where manifest.hjson and manifest.h.tpl reside.
+
+  rom-ext-manifest-generator --output-dir:
+    Directory where manifest.hjson and manifest.h.tpl reside.
```

Comment:
Isn't this where outputs go, not where those inputs are?

Reply:
We'll fix this up afterwards.